src/common/components/ReviewItem/ProfileSchool.tsx

```diff
@@ -1,15 +1,21 @@
+import { type UniversityAbbreviation } from "@prisma/client";
 import { reviewItemTheme, type ReviewItemVariants } from "./ReviewItem.theme";
+import { SchoolIcon } from "@/common/components/CustomIcon";
 
 export type ProfileSchoolProps = ReviewItemVariants & {
   courseCode: string;
+  university: UniversityAbbreviation;
 };
 
-export const ProfileSchool = ({ courseCode }: ProfileSchoolProps) => {
+export const ProfileSchool = ({
+  courseCode,
+  university,
+}: ProfileSchoolProps) => {
   const { schoolContainer, schoolCourseCode, schoolIcon } = reviewItemTheme();
   return (
     <div className={schoolContainer()}>
       <div className={schoolIcon()}>
-        <div className="h-4 w-4 rounded-full bg-red-800"></div>
+        <SchoolIcon school={university} />
```

Comment:
suggestion: could we also add a TODO item at `SchoolIcon` to use `UniversityAbbreviation` as well so it's standardized

tracking this at #106

Reply:
fixed this [here](https://github.com/AfterClass-io/afterclass.io-v2/pull/104/commits/7cdf46559f65df476ed690af89d3dedc6d9716c8), used relative imports for the icons as they are in the same folder level too

Reply:
looks great! 👍